.travis.yml

```
@@ -1,8 +1,32 @@
+conditions: v1
 language: python
-sudo: required
-dist: xenial
-python: "3.7"
-cache: pip
+python:
+  - "3.6"
 before_install:
 - pip install -r requirements.txt
 script: pre-commit run --all-files
+
+jobs:
+  include:
+  - python: 3.7
+
+  - stage: &stage_deploy_name upload new version of python package to PYPI (only for tagged commits)
+    name: Deploy
+    if: tag IS present
+    python: 3.6
```

Use the latest stable.
```suggestion
    python: "3.7"
```